### PR TITLE
Make rc_tests.py meet pylint requirements.

### PR DESCRIPTION
**Pull request by eugulixes**

### CODE REVIEW

File: rc_tests.py [hunk 1]
```diff
@@ -323,89 +350,98 @@ def test_for_pinning_messages(self):
...
         close_button = self.find_by_css(
             '.contextual-bar__header-close.js-close')
 
-        assert len(close_button)
+        assert close_button
 
         close_button.first.click()
 
     def test_visibility_of_pinned_message(self):
+        """Tests if the pinned message is marked as pinned and it's possible to see it. """
```
Comment (polina-popova) [inline rc_tests.py]: It looks like it is a too long string.

File: rc_tests.py [hunk 2]
```diff
@@ -644,38 +696,45 @@ def test_joining_read_only_channel(self):
...
 
         assert members_list.last.text == self.test_username
 
-    # TODO: change the test when https://github.com/RocketChat/Rocket.Chat/issues/11819 is fixed.
     def test_read_only_channel_with_allowed_reacting(self):
-        test_message = self.find_by_css(
-            'div.body.color-primary-font-color ')
+        """Tests if it's not possible to use emojis in the read-only channel
+        where emojis are allowed.
+        Change the test when https://github.com/RocketChat/Rocket.Chat/issues/11819 is closed.
```
Comment (polina-popova) [inline rc_tests.py]: It is also longer then it should be.

File: rc_tests.py [hunk 3]
```diff
@@ -644,38 +696,45 @@ def test_joining_read_only_channel(self):
...
 
         add_reaction = self.find_by_css('.message-actions__button')
 
-        assert not len(add_reaction)
+        assert not add_reaction
 
     def test_read_only_channel_with_disallowed_reacting(self):
+        """Tests if it's possible to use emojis in the read-only channel
+        where emojis are disallowed.
+        Change the test when https://github.com/RocketChat/Rocket.Chat/issues/11819 is closed.
```
Comment (polina-popova) [inline rc_tests.py]: The same comment.

**Comment by eugulixes** [comment]
#39

**Pull request merged**